documentation/assemblies/assembly-iot-service-admin-guide.adoc

```diff
@@ -1,10 +1,17 @@
 // This assembly is included in the following assemblies:
 //
-// assembly-iot-guide.adoc
+// assembly-.adoc
 
 :context: {context}-iot
 
 [id='iot-service-admin-{context}]
-= IoT for service administrators 
+= IoT for service administrators
+
+Service administrators typically install and configure the Internet of Things (IoT) services for {ProductName}.
+
+include::../modules/proc-iot-installing-services.adoc[leveloffset=+1]
+
+include::assembly-iot-configuring-sigfox.adoc[leveloffset=+1]
+
+include::assembly-iot-creating-device.adoc[leveloffset=+1]
```

Comment:
This should go in the tenant guides. Tenants use device registry to manage (create, update, delete) devices